src/boot/efi/devicetree.h

```diff
@@ -9,6 +9,26 @@ struct devicetree_state {
         void *orig;
 };
 
+#define FDT_BEGIN_NODE  0x00000001
+#define FDT_END_NODE    0x00000002
+#define FDT_PROP        0x00000003
+#define FDT_NOP         0x00000004
+#define FDT_END         0x00000009
```

Comment:
given that these are matched against fields of type uint32_t these should be made unsigned constants, i.e. suffixed with `U`.